### ISSUE TITLE
Fully supervised baseline

### ISSUE THREAD

**Issue by voldemortX** [google-research/remixmatch#9]
I don't know much about tensorflow, so I'm a little confused about CIFAR10 fully supervised trainings.
Just to be sure, are you training for 2^(10 + 16) / 50000 = over 1000 epochs (lr 0.002) for the fully supervised baseline?
As I understand that wrn-28-2 is originally trained for 200 epochs (lr 0.1) and can only get to 95.9%-96.0% accuracy on average with autoaugment and cutout. While in your paper it is a little higher than that (96.38%). So is your training schedule better than other people?
Also, do you have the fully-supervised numbers for other labeled ratios (e.g. 4000 labels), it would feel more official and convincing if the numbers are reported by you.

**Comment by carlini** [comment]
Yeah, roughly 1000 epochs. It seems like semi-supervised learning requires more time to train than fully-supervised training. We tried hard to optimize all of our baselines, to make sure that we were comparing against the best of what could be done. For example, we substantially improved the accuracy of VAT in the low-label regime when doing the MixMatch experiments.

I don't think we have any fully-supervised numbers for 4000 labeled examples.

**Comment by voldemortX** [comment]
Thanks for your reply! I'm trying to re-implement this work or FixMatch in PyTorch, maybe I could find a faster training setup in the process.

**Comment by voldemortX** [comment]
Negative, it would seem impossible to train faster without degrading performance. But I do have a fully supervised number on 4000 labeled (CIFAR 10), I can get 86.0% accuracy if I use RA (14 basic op + cutout), N=1, random M.
